content/applications/sales/sales/invoicing/milestone.rst

```diff
@@ -2,32 +2,237 @@
 Invoice project milestones
 ==========================
 
-Milestone invoicing can be used for expensive or large-scale projects,
-with each milestone representing a clear sequence of work that will
-incrementally build up to the completion of the contract. This invoicing
-method is comfortable both for the company which is ensured to get a
-steady cash flow throughout the project lifetime and for the client who
-can monitor the project's progress and pay in several installments.
+Invoicing based on project milestones can be used for expensive or large-scale projects. The series
+of milestones in a project represent a clear sequence of work that will inevitably result in the
+completion of a project and/or contract.
```

Comment:
Don't need the grammatical conjunction here.
The main subject of this section is projects so stick with that 😉 
```suggestion
completion of a project.
```